### ISSUE TITLE
Coverage not displaying in Workspace

### ISSUE THREAD

**Issue by acgritt** [ryanluker/vscode-coverage-gutters#252]
**Describe the bug**
I have a code-workspace file setup with multiple root folders. When I open just the folder with the source I see the coverage just fine. when I use the code-workspace it doesn't display the coverage and just reports this in the output tab:
```
[1580939108663][coverageservice]: INITIALIZING
[1580939870656][coverageservice]: LOADING
[1580939873483][coverageservice]: Loading 1 file(s)
[1580939873484][coverageservice]: Loaded 1 data file(s)
[1580939873492][coverageservice]: Caching 51 coverage(s)
[1580939873492][printDataCoverage]: Coverage -> 51
[1580939873492][coverageservice]: READY
[1580939873492][coverageservice]: RENDERING
[1580939873493][coverageservice]: READY
[1580939874648][coverageservice]: RENDERING
[1580939874649][coverageservice]: READY
```

**Expected behaviour**
Code coverage should highlight the lines in the files.

**Desktop (please complete the following information):**
 - OS: Windows
 - Extension Version: 2.4.3
 - VSCode Version: 1.41.1

**Comment by ryanluker** [comment]
@acgritt thanks for submitting an issue!
Are you able to get the example workspace to work locally?
https://github.com/ryanluker/vscode-coverage-gutters/blob/master/example/example.code-workspace

I wasn't able to reproduce the bug (linux, vscode: 1.41.1, extension: 2.4.3) either through the open workspace or the add folders options.
https://webmshare.com/play/jvXPg

**Comment by acgritt** [comment]
I think I have narrowed down the issue. It seems related to the fact that I have a `name` attribute setup with the folder in the workspace file. so my file looks like this:
```json
{
    "folders": [
        {
            "path": ".",
            "name": "Source"
        },
        {
            "path": "../RG-Definition",
            "name": "ARM"
        }
    ]
}
```

If I remove the `name` property then it is able to highlight the lines fine.

**Comment by ryanluker** [comment]
@acgritt interesting, I will leave this as a bug for the next release (3.0.0) to see if there is something I need to update to better support the workspace file / feature.

**Comment by ryanluker** [comment]
@acgritt This should be fixed in the next release (2.7.3).
Big thanks to @evan-bradley for working on this!
